easylist_cookie/easylist_cookie_specific_hide.txt

```diff
@@ -677,6 +677,7 @@ airnewzealand.eu,przekroj.pl,qv.co.nz,thelittleboxoffice.com##.modal
 airnewzealand.eu,arlingtoncemetery.mil,arthrex.com,axminster.co.uk,daf.be,daf.co.uk,daf.com,daf.fr,daf.global,daf.nl,daf.ru,daf.se,daftrucks.de,daftrucks.dk,daftrucks.it,daftrucks.pl,daftrucks.sk,ferratum.co.uk,fiveguys.co.uk,flipsnack.com,kon-boot.com,lam.co.mz,medtronic.com,primaresearch.org,qv.co.nz##.modal-backdrop
 myanimelist.net##.modal-container
 myanimelist.net##.modal-content
+myanimelist.net##.modal-content-overlay
```

Comment:
Whitout having tested it, but i think you might be able to make it a class rule over a `##.`

```css
myanimelist.net##[class^="modal-content"]
```
Or maybe even better

```css
myanimelist.net##[class^="modal-"]
```

Isn't that a usable solution?

Reply:
`myanimelist.net##.modal-container` is obsolete so `myanimelist.net##div[class^="modal-content"]` will be suffice.